### ISSUE TITLE
Location of temp files for -r

### ISSUE THREAD

**Issue by tseemann** [hsinnan75/MapCaller#71]
I notice when using `-r` that the temporary files are created in the current direcrtory

eg.
```
qqenkendpy.bwt
qqenkendpy.pac
qqenkendpy.ann
qqenkendpy.amb
```

Can you pleae use the system `temddir*()` function?
Normallt that would use `/tmp` or `$TMPDIR` set by the user or system.

**Comment by hsinnan75** [comment]
Yes, they are kept after the whole process terminates, so that users can use the index files for next run. If it is not good, I'll consider to remove them when the process finishes.

**Comment by tseemann** [comment]
Because they have random names, how will the user know where or what they are?

They should use the PODIX temp file functions and be deleted at the end.

These functions are probably in C++ `std` ?
